pkg/cmd/list.go

```diff
@@ -117,13 +120,12 @@ func newListCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.StringVar(&client.TimeFormat, "time-format", "", `format time using golang time formatter. Example: --time-format "2006-01-02 15:04:05Z0700"`)
 	f.BoolVarP(&client.ByDate, "date", "d", false, "sort by release date")
 	f.BoolVarP(&client.SortReverse, "reverse", "r", false, "reverse the sort order")
-	f.BoolVarP(&client.All, "all", "a", false, "show all releases without any filter applied")
-	f.BoolVar(&client.Uninstalled, "uninstalled", false, "show uninstalled releases (if 'helm uninstall --keep-history' was used)")
-	f.BoolVar(&client.Superseded, "superseded", false, "show superseded releases")
-	f.BoolVar(&client.Uninstalling, "uninstalling", false, "show releases that are currently being uninstalled")
-	f.BoolVar(&client.Deployed, "deployed", false, "show deployed releases. If no other is specified, this will be automatically enabled")
-	f.BoolVar(&client.Failed, "failed", false, "show failed releases")
-	f.BoolVar(&client.Pending, "pending", false, "show pending releases")
+	f.BoolVar(&client.Uninstalled, "uninstalled", false, "show only uninstalled releases (if 'helm uninstall --keep-history' was used)")
+	f.BoolVar(&client.Superseded, "superseded", false, "show only superseded releases")
+	f.BoolVar(&client.Uninstalling, "uninstalling", false, "show only releases that are currently being uninstalled")
+	f.BoolVar(&client.Deployed, "deployed", false, "show only deployed releases")
+	f.BoolVar(&client.Failed, "failed", false, "show only failed releases")
+	f.BoolVar(&client.Pending, "pending", false, "show only pending releases")
```

Comment:
If I mix these flags it will list multiple types. For example `--failed --pending` will list both failed and pending releases.

This update added "only" to the description on the flag. Can you remove the "only"?